config/local_info.xml

```diff
@@ -1,9 +1,80 @@
 <?xml version="1.0" encoding="UTF-8"?>
-<schema>
+<config> 
+        <!-- 
+          ## Begin local override.
+          ## Specify a url attribute to be matched  against the incoming requestor
+          ## url. Only a single url is supported.
+          ## If the urls match, the values specified in this section will override
+          ## the default values given in the default (no url) section below.
+          ## Any valid xml element tree can be specified here but it MUST
+          ## exist in the default section that follows.
+          ## Use of elements 'menus','css' and 'name_mappings' are described
+          ## in the defaults section.
+        -->
+        <local_info url="host-172-16-113-107.nubes.stfc.ac.uk">
+
+          <default_filter_by_scope>true</default_filter_by_scope>
+          
+        </local_info>
+        <!-- 
+          ## End local override
+        -->
+
+        <!-- 
+          ## Begin defaults
+        -->
         <local_info>
-        <!-- If read-only is set as true, the portal will prevent changes to the
+
+          <!-- If read-only is set as true, the portal will prevent changes to the
              data by all users. This can be used for a read-only failover instance. -->
-        <read_only>false</read_only>
+          <read_only>false</read_only>
+
+          <!-- menus
+            ## Give the xml element names from config/web_portal/menu.xml
+            ## to select menu link(s) given in the left-hand menu box.
+            ## Selected menus can be hidden by providing an element value
+            ## of any of hide,false,0 or no. The default behaviour is to 
+            ## show the menu subject to the restrictions encoded in menu.xml
+            ## but any other value but those given above will also work to
+            ## enable the menu display.
+          -->
+          <menus>
+            <Projects>true</Projects>
+            <NGI>true</NGI>
+            <AddNGI>show</AddNGI>
+            <AddProject>show</AddProject>
+            <ServiceGroups>true</ServiceGroups>
+            <AddSite>true</AddSite>
+            <Scopes>true</Scopes>
+            <AdminScopes>true</AdminScopes>
+          </menus>
```

Comment:
In this `menus` block, does the use of `true` and `show` illustrate that anything (other than `hide` ,`false`, `0` or `no`) could be used?

I kinda prefer `show`, as this block defines what menus are shown.

Reply:
Yes. Basically anything else shows the menu. Not sure if that's best really. I prefer show and hide and my guess is if you distributed the 'first' version of the file with an example using only these values, they'd become the de facto standard.

Reply:
Indeed, could you use just `show` here then?

Reply:
OK. 
Updated.